ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/SubmitterInfo.scala

```diff
@@ -10,7 +10,12 @@ import java.time.Instant
   * Note that this is used for party-originating changes only. They are
   * usually issued via the Ledger API.
   *
-  * @param submitter: the party that submitted the change.
+  * @param actAs: the set of parties that submitted the change.
```

Comment:
Is this set always guaranteed to be non-empty? If so, I recommend to spell this out.